libraries/AP_Scripting/AP_Scripting_SerialAccess.h

```diff
@@ -14,6 +14,9 @@ class AP_Scripting_SerialAccess {
 
     void begin(uint32_t baud);
 
+    bool configure_parity(uint8_t parity);
+    bool set_stop_bits(uint8_t stop_bits);
```

Comment:
```suggestion
    void configure_parity(uint8_t parity);
    void set_stop_bits(uint8_t stop_bits);
```

And remove the return statements from the implementations. The binding marks these functions as returning `void` and I can't think of a use for the return value so please completely remove it.